integration_tests/features/support/steps.js

```diff
@@ -407,15 +407,12 @@ Then(/(.*) should have (\d+) peers/, async function (nodeName, peerCount) {
   expect(peers.length).to.equal(peerCount);
 });
 
-Then(
-  /(.*) should have at least (\d+) peers/,
-  async function (nodeName, peerCount) {
-    await sleep(500);
-    const client = this.getClient(nodeName);
-    const peers = await client.getPeers();
-    expect(peers.length).to.be.greaterThanOrEqual(peerCount);
-  }
-);
+Then(/(.*) has at least (\d+) peers/, async function (nodeName, peerCount) {
+  await sleep(500);
```

Comment:
This commit only changed some of the wording. This means this is most likely a flaky test. Should we not remove this sleep and put a longer wait and test till the condition matches in rather?
Something like this:
`await waitFor(
        async () => {`

Reply:
The actual error on CI was a port conflict, pointing to a problem in the way we assign open ports in general, not related to the test but I agree we should allow more than 500ms for the peers to sync so I've added get_peer polling as  suggested.